### PR TITLE
Add option to switch back to SSH REX mode

### PR DESCRIPTION
**Pull request by adamlazik1**
BZ: https://bugzilla.redhat.com/show_bug.cgi?id=2269398

* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.10/Katello 4.12
* [x] Foreman 3.9/Katello 4.11 (planned Satellite 6.15)
* [x] Foreman 3.8/Katello 4.10
* [x] Foreman 3.7/Katello 4.9 (Satellite 6.14)
* [x] Foreman 3.6/Katello 4.8
* [x] Foreman 3.5/Katello 4.7 (Satellite 6.13; orcharhino 6.6/6.7)
* [x] Foreman 3.4/Katello 4.6 (EL8 only)
* [x] Foreman 3.3/Katello 4.5 on EL7 & EL8 (Satellite 6.12 on EL8 only; orcharhino 6.4/6.5 on EL8 only)
* We do not accept PRs for Foreman older than 3.3.

### CODE REVIEW

**Review by adamruzicka — APPROVED** [review]
This is the way

File: guides/common/modules/con_transport-modes-for-remote-execution.adoc [hunk 1]
```diff
@@ -9,6 +9,16 @@ The SSH service must be enabled and active on the target hosts.
 The remote execution {SmartProxy} must have access to the SSH port on the target hosts.
 Unless you have a different setting, the standard SSH port is 22.
 
+[NOTE]
+====
+If your {SmartProxy} already uses the `pull-mqtt` mode and you want to switch back to the `ssh` mode, run this {foreman-installer} command:
+
+[options="nowrap",subs="+quotes,verbatim,attributes"]
+----
+{foreman-installer} --foreman-proxy-plugin-remote-execution-script-mode=ssh
```
Comment (maximiliankolb) [inline guides/common/modules/con_transport-modes-for-remote-execution.adoc:18]:
```suggestion
# {foreman-installer} --foreman-proxy-plugin-remote-execution-script-mode=ssh
```

File: guides/common/modules/con_transport-modes-for-remote-execution.adoc [hunk 2]
```diff
@@ -9,6 +9,16 @@ The SSH service must be enabled and active on the target hosts.
 The remote execution {SmartProxy} must have access to the SSH port on the target hosts.
 Unless you have a different setting, the standard SSH port is 22.
 
+[NOTE]
+====
+If your {SmartProxy} already uses the `pull-mqtt` mode and you want to switch back to the `ssh` mode, run this {foreman-installer} command:
```
Comment (maximiliankolb) [inline guides/common/modules/con_transport-modes-for-remote-execution.adoc:14]:
```suggestion
If your {SmartProxy} already uses the `pull-mqtt` mode and you want to switch back to the `ssh` mode, run this `{foreman-installer}` command:
```

File: guides/common/modules/con_transport-modes-for-remote-execution.adoc (hunk 1 above)
Comment (adamlazik1) [reply guides/common/modules/con_transport-modes-for-remote-execution.adoc]: Applied.

File: guides/common/modules/con_transport-modes-for-remote-execution.adoc (hunk 2 above)
Comment (adamlazik1) [reply guides/common/modules/con_transport-modes-for-remote-execution.adoc]: Applied.

**Review by maximiliankolb — APPROVED** [review]
LGTM. I wonder if it would be better to move this to a separate procedure; but I'm also OK with this admonition.

**Comment by adamlazik1** [comment]
Seemed too small for me to move it into a separate module so I'd prefer the admonition.

**Comment by asteflova** [comment]
> Seemed too small for me to move it into a separate module so I'd prefer the admonition.

I haven't looked at the context here, but [RH-SSG](https://redhat-documentation.github.io/supplementary-style-guide/#admonitions) says this:

> Do not include procedures in an admonition.

This does look like a procedure.

**Comment by adamlazik1** [comment]
@asteflova should I revert this? The reason I don't want this as a procedure is because I think it would increase the doc bloat.

**Comment by asteflova** [comment]
@adamlazik1 Nah, don't worry about it :) Looking at the whole module, it would require a more significant rewrite to get it aligned with mod docs and style guides (it's like the module can't decide whether it wants to be a concept or a procedure, which makes its focus unclear). So even though the note does not exactly follow RH-SSG, I don't believe it's worth revising this bug fix. Thanks for working on this!

**Comment by adamlazik1** [comment]
Merged and cherry-picked:
   462868d96c..613b374394  3.10 -> 3.10
   b98d7a4ee1..da78e096b6  3.3 -> 3.3
   a012be17e3..bf1f600757  3.4 -> 3.4
   737b484b1e..ce7ade743c  3.5 -> 3.5
   2e3c03391a..4be20f8a3e  3.6 -> 3.6
   0a0211e6fc..22de68986a  3.7 -> 3.7
   82d9879d59..5612cf3728  3.8 -> 3.8
   39a71ff6f2..292e9d7577  3.9 -> 3.9

**Pull request merged**